public/index.html

```diff
@@ -3,9 +3,11 @@
 <head>
   <meta charset="UTF-8">
   <title>Calendar</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
i think we're not supposed to use a general style sheet and just use css modules or styled components instead?

Reply:
I'm doing this first thing tomo! Did you use css modules? Or what's the other way using react? Either way, shouldn't be too hard, I hope!

Reply:
i used styled components